scripts/us_epa/national_emissions_inventory/process.py

```diff
@@ -0,0 +1,336 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This Python Script Load the datasets, cleans it
+and generates cleaned CSV, MCF, TMCF file.
+"""
+
+import os
+import sys
+from sre_constants import IN
+from absl import app, flags
+import pandas as pd
+import numpy as np
+
+sys.path.insert(
+    1, os.path.join(os.path.dirname(os.path.abspath(__file__)), '../../../'))
+
+from util.statvar_dcid_generator import get_statvar_dcid
+
+sys.path.insert(1, os.path.dirname(os.path.abspath(__file__)))
+
+from config import *
+
+FLAGS = flags.FLAGS
+default_input_path = os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                                  "input_files")
+flags.DEFINE_string("input_path", default_input_path, "Import Data File's List")
+
+_MCF_TEMPLATE = ("Node: dcid:{statvar}\n"
+                 "name: \"Annual Amount Emissions {statvar_name}\"\n"
+                 "typeOf: dcs:StatisticalVariable\n"
+                 "populationType: dcs:Emissions\n"
+                 "measurementQualifier: dcs:Annual{scc}"
```

Comment:
missing "\n" at line 44?

Reply:
We are not adding "\n" at line 44 as not every node has emittedThing property. Hence, we are adding "\n" separately.